docs/python.md

```diff
@@ -0,0 +1,17 @@
+## Description
```

Comment:
Add information that this is not recommended.

Reply:
Better now? :)